packages/webview_flutter/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -94,34 +94,46 @@ - (instancetype)initWithFrame:(CGRect)frame
     [self updateAutoMediaPlaybackPolicy:args[@"autoMediaPlaybackPolicy"]
                         inConfiguration:configuration];
 
-    _webView = [[FLTWKWebView alloc] initWithFrame:frame configuration:configuration];
     _navigationDelegate = [[FLTWKNavigationDelegate alloc] initWithChannel:_channel];
-    _webView.UIDelegate = self;
-    _webView.navigationDelegate = _navigationDelegate;
+    _webView = [self createFLTWKWebViewWithFrame:frame
```

Comment:
Do not call methods in init: https://google.github.io/styleguide/objcguide.html#avoid-messaging-the-current-object-within-initializers-and--dealloc

I see that there are already violations of this; we should not add more (and should at some point fix those).